Assets/SEE/Controls/Actions/ContextMenuAction.cs

```diff
@@ -325,10 +669,209 @@ void ShowUnifiedDiff()
                 ActivateWindow(ShowCodeAction.ShowUnifiedDiff(gameObject.MustGetComponent<EdgeRef>()));
             }
 
+        }
+
+        /// <summary>
+        /// Returns the options available for the given edge.
+        /// </summary>
+        /// <param name="popupMenu">The popup menu in which the options should be displayed.</param>
+        /// <param name="position">The position to be displayed the popup menu.</param>
+        /// <param name="raycastHitPosition">The position of the raycast hit.</param>
+        /// <param name="edge">The edge to get the options for</param>
+        /// <param name="gameObject">The game object that the edge is attached to</param>
+        /// <param name="appendActions">Options to be append at the end of the entries.</param>
+        /// <returns>Options available for the given edge</returns>
+        private static IEnumerable<PopupMenuEntry> GetEdgeOptions
+            (PopupMenu popupMenu,
+            Vector3 position,
+            Vector3 raycastHitPosition,
+            Edge edge,
+            GameObject gameObject,
+            IEnumerable<PopupMenuAction> appendActions = null)
+        {
+            IList<PopupMenuEntry> actions = new List<PopupMenuEntry>();
+
+            if (edge.IsInImplementation() && ReflexionGraph.IsDivergent(edge))
+            {
+                actions.Add(new PopupMenuAction("Accept Divergence", AcceptDivergence, Icons.Checkmark));
+            }
+
+            List<PopupMenuEntry> entries = new();
+            if (actions.Count > 0)
```

Comment:
Because we currently only have AcceptDivergence as an EdgeOption, and an empty Edge Options should be hidden. An empty context menu doesn't make sense.

Reply:
Got it, thanks!